contrib/src/main/java/org/archive/crawler/frontier/AMQPUrlReceiver.java

```diff
@@ -152,6 +151,28 @@ public void setForceFetch(boolean forceFetch) {
         this.forceFetch = forceFetch;
     }
 
+    /**
+     * The maximum prefetch count to use, meaning the maximum number of messages
+     * to be consumed without being acknowledged. Use 'null' to specify there
+     * should be no upper limit (the default).
+     */
+    private Integer prefetchCount = null;
```

Comment:
It doesn't need to be configurable, I just assumed that would be preferred. A default of e.g. 1000 or something would prevent the problems we've seen.